appendices/versions.txt

```diff
@@ -188,37 +188,70 @@ number of key extensions into the core API:
 
 include::{generated}/meta/promoted_extensions_VK_VERSION_1_1.txt[]
 
-The only changes to the functionality added by these extensions were to
-`VK_KHR_shader_draw_parameters`, which had the
-<<features-shaderDrawParameters, pname:shaderDrawParameters>> feature bit
-added to determine support in the core API, and
-<<features-variablePointersStorageBuffer,
-pname:variablePointersStorageBuffer>> from `VK_KHR_variable_pointers` was
-made optional.
+All differences in behavior between these extensions and the corresponding
+Vulkan 1.1 functionality are summarized below.
 
-[[versions-1.1-new-features]]
-Additionally, Vulkan 1.1 added support for <<shaders-group-operations, group
-operations>> with <<shaders-scope-subgroup, subgroup scope>>,
-<<memory-protected-memory, protected memory>>, and a new command to
-<<vkEnumerateInstanceVersion, enumerate the instance version>>.
+=== Differences relative to `VK_KHR_16bit_storage`
+
+If the `<<VK_KHR_16bit_storage>>` extension is not supported, support for
+the <<features-storageBuffer16BitAccess,storageBuffer16BitAccess>> feature
+is optional.
+Support for this feature is defined by
+slink:VkPhysicalDevice16BitStorageFeatures::pname:storageBuffer16BitAccess
+ifdef::VK_VERSION_1_2[]
+or slink:VkPhysicalDeviceVulkan11Features::pname:storageBuffer16BitAccess
+endif::VK_VERSION_1_2[]
+when queried via flink:vkGetPhysicalDeviceFeatures2.
 
 === Differences relative to `VK_KHR_sampler_ycbcr_conversion`
 
 If the `<<VK_KHR_sampler_ycbcr_conversion>>` extension is not supported,
 support for the <<features-samplerYcbcrConversion,samplerYcbcrConversion>>
 feature is optional.
 Support for this feature is defined by
-slink:VkPhysicalDeviceVulkan11Features::pname:samplerYcbcrConversion when
-queried via flink:vkGetPhysicalDeviceFeatures2.
+slink:VkPhysicalDeviceSamplerYcbcrConversionFeatures::pname:samplerYcbcrConversion
+ifdef::VK_VERSION_1_2[]
```

Comment:
This ifdef added is wrong as `or slink:VkPhysicalDeviceVulkan11Features::pname:samplerYcbcrConversion` is known with a 1.1 version of the spec

Also the reason for not having the original feature is this section is describing the "Differences relative to VK_KHR_sampler_ycbcr_conversion" so I don't see the reason to add both features

Reply:
Is it? `VkPhysicalDeviceVulkan11Features` was added in Vulkan 1.2, so I should not be referencing it in 1.1.

Reply:
ok, so it was added to the spec in the 1.2 timeframe, but if I load 
https://www.khronos.org/registry/vulkan/specs/1.1-extensions/html/vkspec.html
it is there

Think of it as:

it was not in 1.1.`N`, but it is in 1.1.`N+1`

Reply:
nvm, that is totally wrong and breaks backward compatibility... sorry, you are totally correct on this!